extensions/toggl-track/src/index.tsx

```diff
@@ -11,31 +11,40 @@ import {
   showToast,
   SubmitFormAction,
   ToastStyle,
+  getPreferenceValues,
 } from "@raycast/api";
 import { TimeEntry } from "./toggl/types";
 import toggl from "./toggl";
 import { storage } from "./storage";
 import ProjectListItem from "./components/ProjectListItem";
+import StartTimeEntryForm from "./components/StartTimeEntryForm";
 import CreateTimeEntryForm from "./components/CreateTimeEntryForm";
+import EditTimeEntryForm from "./components/EditTimeEntryForm";
+
+interface Preferences {
+  allowDelete: boolean;
+}
+
+const preferences: Preferences = getPreferenceValues();
 
 dayjs.extend(duration);
 
 function ListView() {
   const { isLoading, isValidToken, projectGroups, runningTimeEntry, timeEntries, projects } = useAppContext();
   const getProjectById = (id: number) => projects.find((p) => p.id === id);
-
   const timeEntriesWithUniqueProjectAndDescription = timeEntries.reduce((acc, timeEntry) => {
-    const existing = acc.find((t) => t.description === timeEntry.description && t.pid === timeEntry.pid);
+    const existing = acc.find((t) => (t.description === timeEntry.description && t.pid === timeEntry.pid) || !("stop" in timeEntry));
```

Comment:
Not sure what `"stop" in timeEntry` solves here?

Reply:
It stops the current running time entry from showing up under recent time entries, so it doesn't appear twice.